.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "cSpell.words": [
```

Comment:
:) 
I don't think that it should be here.You can add words to user settings dictionary